src/util/common.ts

```diff
@@ -1,9 +1,20 @@
 // common utilities and helper functions
 
-import { BigDecimal, BigInt, log } from '@graphprotocol/graph-ts';
+import { Address, BigDecimal, BigInt, log } from '@graphprotocol/graph-ts';
+
+import { User } from '../../generated/schema'
+import { ZERO_BIG_INT, ZERO_BIG_DECIMAL } from '../util/constants'
 
 
 export function integerToDecimal(value: BigInt, decimals: BigInt = BigInt.fromI32(18)): BigDecimal {
   let denom = BigInt.fromI32(10).pow(decimals.toI32() as u8);
   return value.toBigDecimal().div(denom.toBigDecimal());
 }
+
+export function initializeUser(address: Address): User {
```

Comment:
can we name this to be consistent with `createNewToken()`? (feel free to rename that one instead if you prefer "initialize")

Reply:
Good call, I'll stick with 'createNew'